aptos-move/framework/aptos-framework/sources/delegation_pool.move

```diff
@@ -635,12 +643,23 @@ module aptos_framework::delegation_pool {
         let pool = borrow_global_mut<DelegationPool>(pool_address);
         let delegator_address = signer::address_of(delegator);
 
+        spec {
+            update pre_inactive_shares_OLC_delegator = pool_u64::spec_balance(table::spec_get(pool.inactive_shares, pool.observed_lockup_cycle), delegator_address);
+            update pre_active_shares_OLC_delegator = pool_u64::spec_balance(pool.active_shares, delegator_address);
+        };
+
         amount = coins_to_transfer_to_ensure_min_stake(
             &pool.active_shares,
             pending_inactive_shares_pool(pool),
             delegator_address,
             amount,
         );
+
+        spec{
+            update ghost_source_pool = pool.active_shares;
```

Comment:
Could we use a bit more descriptive variable name without the prefix "ghost" like the other variables above?

We can discuss and decide on the convention for ghost variables offline.